rank_llm/retrieve_and_rerank.py

```diff
@@ -0,0 +1,115 @@
+import json
+from pathlib import Path
+from typing import List, Union, Dict, Any
+
+from rank_llm.rank_gpt import SafeOpenai
+from rank_llm.rankllm import PromptMode
+from rank_llm.rank_vicuna import RankVicuna
+from rank_llm.trec_eval import EvalFunction
+from rank_llm.pyserini_retriever import RetrievalMethod
+from rank_llm.retriever import Retriever, RetrievalMode
+from rank_llm.reranker import Reranker
+from rank_llm.topics_dict import TOPICS
+
+
+def get_api_key() -> str:
+    from dotenv import dotenv_values, load_dotenv
+    import os
+
+    load_dotenv(dotenv_path=f"../.env.local")

```

Comment:
```suggestion
    load_dotenv(dotenv_path=f".env.local")

```

Reply:
Moving to parent dir is no longer needed

Reply:
Updated